crates/typst-library/src/model/list.rs

```diff
@@ -67,13 +67,13 @@ pub struct ListElem {
     #[default(true)]
     pub tight: bool,
 
-    /// The marker which introduces each item.
+    /// 各項目の先頭に付ける記号。
     ///
-    /// Instead of plain content, you can also pass an array with multiple
-    /// markers that should be used for nested lists. If the list nesting depth
-    /// exceeds the number of markers, the markers are cycled. For total
-    /// control, you may pass a function that maps the list's nesting depth
-    /// (starting from `{0}`) to a desired marker.
+    /// 単純なコンテンツの代わりに、ネストされたリストに使用する、
+    /// 複数の記号を持つ配列を渡すこともできます。
+    /// リストのネストの深さが記号の数を超えた場合、使用される記号は循環します。
+    /// 完全な制御を行いたい場合は、
+    /// リストのネストの深さ（`{0}`から開始する）を希望する記号にマッピングする関数を渡すことができます。
```

Comment:
【提案】記号→マーカー
【提案】「制御を行いたい」→「制御したい」
【提案】マッピングを意訳（案）

```suggestion
    /// 単純なコンテンツの代わりに、ネストされたリストに使用する、
    /// 複数のマーカーを持つ配列を渡すこともできます。
    /// リストのネストの深さがマーカーの数を超えた場合、使用されるマーカーは循環します。
    /// 完全に制御したい場合は、
    /// リストのネストの深さ（`{0}`から開始する）に応じて、使用するマーカーを決める関数を渡すこともできます。
```